guides/v2.2/cloud/live/live-sanity-check.md

```diff
@@ -28,7 +28,7 @@ If you modified your `composer.json` file to add modules, we recommend running t
 Your Git branch must have the following files for building and deploying for your local and to Integration, Staging, and Production environments:
 
 * `auth.json` in the root Magento directory. This file includes the Magento authentication keys entered when creating the project. If you need to verify the file and settings, see [Troubleshoot deployment]({{ page.baseurl }}/cloud/trouble/troubleshoot-deployment.html).
-* `config.php` if you used [Configuration Management]({{ page.baseurl }}/cloud/live/sens-data-over.html) for 2.2.X
+* `config.php` if you used [Configuration Management]({{ page.baseurl }}/cloud/live/sens-data-over.html)
```

Comment:
This concept is old and only refers to upgrading from an older version of Magento. We may still need it, but I'm not 100% on that.

Reply:
Here's some content about it. Could also provide link, maybe: https://devdocs.magedevteam.com/923/guides/v2.3/cloud/project/project-upgrade.html#configuration-management